pkg/config/template.go

```diff
@@ -345,6 +345,18 @@ allowed_annotations = [
 # Kata Containers with the Firecracker VMM
 #[crio.runtime.runtimes.kata-fc]
 
+#[crio.runtime.workloads.workload-type]
+# cpu_set = "0-1"
+# label = "workload-type.crio.io"
+# cpu_shares_annotation = "cpus.workload-type.crio.io"
+# Where:
+# cpu_set is the CPU set for the workload type
+# label is used to identify the workload type
```

Comment:
Can we remove the label and only use the `cpu_shares_annotation` for identifying that the workload option should apply?

Reply:
I think it could be useful to have a generic label `workload-type.crio.io` and a supported_labels slice `["cpus"]` 

then the full annotation would be constructed as:
```go
for _, label := range SupportedLabels {
 checkForAnnotation(container, label + genericLabel + "/" + ctr.Name()
}
```